src/services/security/modules/ApprovalModule/index.ts

```diff
@@ -52,11 +60,59 @@ export class ApprovalModule implements SecurityModule<ApprovalModuleRequest, App
     return []
   }
 
-  scanTransaction(request: ApprovalModuleRequest): SecurityResponse<ApprovalModuleResponse> {
-    const { safeTransaction } = request
-    const safeTxData = safeTransaction.data.data
+  private static scanPermitDetails(details: PermitDetails): Pick<Approval, 'amount' | 'tokenAddress'> {
```

Comment:
Would it make more sense to name this `getPermitDetails`? Its not really scanning something.